lib/modules/platform/gitlab/index.ts

```diff
@@ -321,7 +325,7 @@ export async function initRepo({
   endpoint,
   includeMirrors,
 }: RepoParams): Promise<RepoResult> {
-  config = {} as any;
+  config ??= {} as any;
```

Comment:
@viceice Should I separate this out in a dffierent PR?

Because of this the config gets reset which causes the `author` set during `initPlatform` to be lost after `initRepo`

Reply:
no, use a separate variable for username, like we do on other plarforms.